### ISSUE TITLE
[a11y audit] Testing: Chatbot MHV Prescription Skill: Plain Language 4.2.4 - 4.2.8

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#67068]
## Chatbot Audit 
[**Google Sheet**: Chatbot Audit](https://docs.google.com/spreadsheets/d/12NgigOQTKgsSexnq_ioUwzXQLtRGRcrIYN--yb3i2Ig/edit#gid=1502697987)

## Testing Info.
- Testing should be done in the dev environment, per @coforma-terry on 101/6/2023
    - https://dev.va.gov/contact-us/virtual-agent/
- Use this [endpoint](https://dev.va.gov/contact-us/virtual-agent/) and test user 228 from [this page](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-cst-users.csv). 
- Use “refills” or “prescriptions” to get the flow started.
-  **Additional Trigger:** "refill prescription"
### Authentication System Update
`SS added: 10/9/2023 c/o Eli, via Karen on the Chatbot Team`
- Testing an authenticated user in chatbot:
    - `DO NOT` use the header sign-in button.
    - `DO` follow the login flow baked into the chatbot.
> _The biggest bit of note is that the authentication system used in dev vs prod is fundamentally different, so, if you need to test an authenticated thing in the chatbot do not use the big “sign in” button in the header of the page where it is embeded. Instead, follow the login flow that is baked into the chatbot itself._

```[tasklist]
### Tasks
- [x] 4.2.4 [Pass] Keep messages short (criteria not totally relevant to RX)
- [x] 4.2.5 [See notes] Use simple messages
- [x] 4.2.6 [Pass] Limit the number of messages
- [x] 4.2.7 [Pass] Build simple conversations
- [x] 4.2.8 [Further investigation needed] Clarify complex topics
```

```[tasklist]
### Tasks
```

**Comment by eli-oat** [comment]
A note of record to elaborate with further investigation: 

- is there a way to refer to medications directly? 
- if there is a way to refer to a medication directly is there a map for selection, e.g. rather than having to say or type the medication's name can you say "first medication" or a more common name?
- I have noticed the construction "First..., next..., last." How is this construct deployed for lists of dozens or hundreds of medications? Is "next" repeated many times, or are numbers used?

**Comment by sara-amanda** [comment]
*Excellent points! I went through my notes, and it appears Brian made a
similar inquiry during Midpoint Numerical Values. Also, sharing info about
the vendor and some restrictions in place currently *

*Midpoint a11y Feedback*




*c/o Brian (a11y)In one flow in Sketch, the list of active prescriptions
are ordered with numerical values
<https://www.sketch.com/s/b5a58a23-6128-421b-9869-50bad5b2686f/a/mPbGzKg>,
but in another flow those numerical values aren't displayed
<https://www.sketch.com/s/b5a58a23-6128-421b-9869-50bad5b2686f/a/Mme0Jk7>.
Could you clarify whether those numbers will be present? And if so, will
the chatbot recognize a response of "three" (spoken or typed) as equivalent
to "acetaminophen"? From an accessibility perspective, I think there would
be a ton of value in providing that numerical backup, as some users may
struggle to say medication names cleanly enough for the speech recognition
to get a good match.Source: Midpoint review*

*Content Source *

This RX feature is an external feature being ported into the chatbot. The
content is coming from another vendor called RAIN group, which is a sub to
Iron Bow. This Rx feature is a project of the Office of Connected Care at
VHA.

Chatbot team doesn’t directly control the content for this RX skill, it’s
coming in from the external VHA vendor

General note: Specific content and data questions & suggestions will have
to go to RAIN vendor/VHA

How "active" status works in this feature

This feature says it’s only showing "active" prescriptions ready for
refill, but MHV/mobile app/new medications tool all have several types of
"active" prescriptions — including 1 (Active: On hold) that you can’t refill

From this initial conversation it sounds like this feature is only pulling
in "Active" prescriptions with no substatus, not any "Active: Parked"
"Active: On hold" "Active: Refill in process" or "Active: Submitted"
statuses, but we’ll need to confirm this

Note: We haven’t yet seen screens showing the "track" and "review all
active prescriptions" options in the Rx feature, only the "refill" option

Source: CAIA Sync 9/19/2023

*Design Intent Notes*

Erin (IA): Is there a way for individuals to understand the prescriptions
in human words?

Is there a way to nudge or remind people to understand if they have
multiple prescriptions which one it is? Differentiation.

Nathalie, doesn’t know the answer to this? There are some data query
concerns, I believe. Not sure if that is within the realms of
possibilities.

Source: Deign Intent

**Comment by sara-amanda** [comment]
## Re: Questions and items for further investigation 10/9/2023 

**Excellent points! @eli-oat**  So, I went through my notes, and it appears Brian made a similar inquiry during Midpoint about `Numerical Values`. Also, sharing `info about the vendor` and `some restrictions in place currently`.
 
### Feedback from Brian (a11y)

> In one flow in Sketch, the list of active prescriptions are ordered with numerical values, but in another flow those numerical values aren't displayed. Could you clarify whether those numbers will be present? And if so, will the chatbot recognize a response of "three" (spoken or typed) as equivalent to "acetaminophen"? 
> 
> From an accessibility perspective, I think there would be a ton of value in providing that numerical backup, as some users may struggle to say medication names cleanly enough for the speech recognition to get a good match.

`Source: Midpoint Review`

### Content Source 

> This RX feature is an external feature being ported into the chatbot. The content is coming from another vendor called RAIN group, which is a sub to Iron Bow. This Rx feature is a project of the Office of Connected Care at VHA.
> Chatbot team doesn’t directly control the content for this RX skill, it’s coming in from the external VHA vendor

### General note: 

> Specific content and data questions & suggestions will have to go to RAIN vendor/VHA


### How "active" status works in this feature

> This feature says it’s only showing "active" prescriptions ready for refill, but MHV/mobile app/new medications tool all have several types of "active" prescriptions — including 1 (Active: On hold) that you can’t refill

> From this initial conversation it sounds like this feature is only pulling in "Active" prescriptions with no substatus, not any "Active: Parked" "Active: On hold" "Active: Refill in process" or "Active: Submitted" statuses, but we’ll need to confirm this

> **Note**: We haven’t yet seen screens showing the "track" and "review all active prescriptions" options in the Rx feature, only the "refill" option

`Source: CAIA Sync 9/19/2023`

### Feedback from Erin (IA)

> Is there a way for individuals to understand the prescriptions in human words? 
> 
> Is there a way to nudge or remind people to understand if they have multiple prescriptions which one it is? Differentiation. 

> Nathalie, doesn’t know the answer to this? There are some data query concerns, I believe. Not sure if that is within the realms of possibilities. 

`Source: Design Intent`

**Comment by sara-amanda** [comment]
Changing status from `open` to `closed`. Ticket completed 3/13/2023.
